Dockerfile

```diff
@@ -3,6 +3,7 @@
 ARG CROSS="false"
 ARG GO_VERSION=1.13.1
 ARG DEBIAN_FRONTEND=noninteractive
+ARG VPNKIT_DIGEST=e508a17cfacc8fd39261d5b4e397df2b953690da577e2c987a47630cd0c42f8e
```

Comment:
This doesn't seem cross platform.

Reply:
Oh! Now I recall why we did it the way it was 😞

Reply:
Actually, I think that should work, as we currently also pull it unconditionally?

Reply:
But the image is for a specific architecture (amd64, I'm assuming).

Reply:
I guess it's not really a change from what we already have.

Reply:
Yes, looks like we worked around it at the time by naming the binary `something.x86`. I guess with BuildKit we could make an architecture-specific dummy stage, but not sure if that would make things a lot cleaner